### ISSUE TITLE
zero accessibility

### ISSUE THREAD

**Issue by phazei** [FortAwesome/react-native-fontawesome#177]
Considering how good your accessibility support is in React 18 web, I'm very disappointed to see that your component completely lacks all accessibility.  It doesn't appear to accept the standard accessibility react props.  please fix.
